packages/bpk-docs/src/pages/BadgePage/BadgePage.js

```diff
@@ -83,6 +85,117 @@ const components = [
+        Bananas
+      </BpkBadge>,
+      <BpkBadge type={BADGE_TYPES.SUCCESS} className={badgeClassName}>
+        Strawberries
+      </BpkBadge>,
+      <BpkBadge type={BADGE_TYPES.SUCCESS} className={badgeClassName}>
+        Pears
+      </BpkBadge>,
+    ],
+  },
+  {
+    id: 'destructive',
+    title: 'Destructive',
+    blurb: [
+      <Paragraph>
+        The destructive badge is a red color as shown below.
+      </Paragraph>,
+    ],
+    examples: [
+      <BpkBadge type={BADGE_TYPES.DESTRUCTIVE} className={badgeClassName}>
+        Apples
+      </BpkBadge>,
+      <BpkBadge type={BADGE_TYPES.DESTRUCTIVE} className={badgeClassName}>
+        Bananas
+      </BpkBadge>,
+      <BpkBadge type={BADGE_TYPES.DESTRUCTIVE} className={badgeClassName}>
+        Strawberries
+      </BpkBadge>,
+      <BpkBadge type={BADGE_TYPES.DESTRUCTIVE} className={badgeClassName}>
+        Pears
+      </BpkBadge>,
+    ],
+  },
+  {
+    id: 'light',
+    title: 'Light',
+    blurb: [
+      <Paragraph>The light badge is a gray color as shown below.</Paragraph>,
+    ],
+    examples: [
+      <BpkBadge type={BADGE_TYPES.LIGHT} className={badgeClassName}>
+        Apples
+      </BpkBadge>,
+      <BpkBadge type={BADGE_TYPES.LIGHT} className={badgeClassName}>
+        Bananas
+      </BpkBadge>,
+      <BpkBadge type={BADGE_TYPES.LIGHT} className={badgeClassName}>
+        Strawberries
+      </BpkBadge>,
+      <BpkBadge type={BADGE_TYPES.LIGHT} className={badgeClassName}>
+        Pears
+      </BpkBadge>,
+    ],
+  },
+  {
+    id: 'inverse',
+    title: 'Inverse',
+    blurb: [<Paragraph>The inverse badge is white as shown below.</Paragraph>],
+    examples: [
+      <BadgeLayout>
+        <BpkBadge type={BADGE_TYPES.INVERSE} className={badgeClassName}>
+          Apples
+        </BpkBadge>
+        <BpkBadge type={BADGE_TYPES.INVERSE} className={badgeClassName}>
+          Bananas
+        </BpkBadge>
+        <BpkBadge type={BADGE_TYPES.INVERSE} className={badgeClassName}>
+          Strawberries
+        </BpkBadge>
+        <BpkBadge type={BADGE_TYPES.INVERSE} className={badgeClassName}>
+          Pears
+        </BpkBadge>
+      </BadgeLayout>,
+    ],
+  },
+  {
+    id: 'outline',
+    title: 'Outline',
+    blurb: [
+      <Paragraph>
+        The outline badge has a somewhat transparent background as shown below.
+      </Paragraph>,
+    ],
+    examples: [
+      <BadgeLayout>
+        <BpkBadge type={BADGE_TYPES.OUTLINE} className={badgeClassName}>
+          Apples
+        </BpkBadge>
+        <BpkBadge type={BADGE_TYPES.OUTLINE} className={badgeClassName}>
+          Bananas
+        </BpkBadge>
+        <BpkBadge type={BADGE_TYPES.OUTLINE} className={badgeClassName}>
+          Strawberries
+        </BpkBadge>
+        <BpkBadge type={BADGE_TYPES.OUTLINE} className={badgeClassName}>
+          Pears
+        </BpkBadge>
+      </BadgeLayout>,
+    ],
+  },
 ];
 
 const BadgePage = () => (
```

Comment:
GH won't let me add a comment on the correct line as it's not part of the diff, but on line 205 with the page blurb, it mentions that badges are only available in one colour. This should be removed.

Reply:
Done 👍